tunnel/wstuncli.go

```diff
@@ -155,6 +160,33 @@ func NewWSTunnelClient(args []string) *WSTunnelClient {
 		wstunCli.Proxy = proxyURL
 	}
 
+	if *cliport != "" {
+		portList := strings.Split(*cliport, ",")
+		var ClientPorts []string
```

Comment:
This variable doesn't need to be exported (i.e) it can be `clientPorts`. Also it can be a slice of int as we know the ports are numbers.